### PR TITLE
Move release logic for vendoring deps to version history

### PR DESCRIPTION
**Pull request by jul-sh**
Do we still need this to run on each PR? It seems like this may have been superseded by transparent release.

### CODE REVIEW

**Comment by jul-sh** [comment]
Vendoring deps could make sense still, but if we want that we should probs include that in the transparent release process.

**Review by tiziano88 — APPROVED** [review]

File: scripts/release_build_oak_functions [hunk 1]
```diff
@@ -1,14 +0,0 @@
-#!/usr/bin/env bash
```
Comment (rbehjati) [inline scripts/release_build_oak_functions:1]: This is used in our internal cloudbuild script for release. Is there a plan for updating that as well?

Comment (jul-sh) [reply scripts/release_build_oak_functions:1]: Created cl/430414831 as a minimal update to internal references. There's more details in the CL description, we can discuss it there. :)

Comment (rbehjati) [reply scripts/release_build_oak_functions:1]: Thanks :)

**Review by rbehjati — APPROVED** [review]

**Pull request merged**